### ISSUE TITLE
Status Code: 403; Forbidden after deploying to production (on azure linux app-service)

### ISSUE THREAD

**Issue by Axendo** [nul800sebastiaan/Cultiv.Hangfire#5]
Hi @nul800sebastiaan ,

Today I encountered an issue after deploying a site with this package to an azure app-service environment, hangfire seemed to be running but the dashboard gave error Status Code: 403; Forbidden.

After quite some digging arround I came to the conclusion that the AuthourizedUmbracoDashboard does authorize access to the endpoint, but hangfire still blocks it because it doesn't have an Authorization configured.

I solved this by adding a dummy IDashboardAuthorizationFilter implementation that simply returns **true** on every request, not being logged in as a backoffice user with access to the settings section still disables dashboard access by way of the AuthorizationPolicy in your package.

Fixing is simple, just add a class;
```
public class UmbracoAuthorizationFilter : IDashboardAuthorizationFilter
{
  public bool Authorize(DashboardContext context) 
  {
    return true;
  }
}
```
and change the endpoint configuration in UmbracoPipelineFilter to 
```
// Add the dashboard and make sure it's authorized with the named policy above
builder.Services.Configure<UmbracoPipelineOptions>(options =>
{
    options.AddFilter(new UmbracoPipelineFilter(Constants.System.HangfireDashboard)
    {
        Endpoints = app => app.UseEndpoints(endpoints =>
        {
            endpoints.MapHangfireDashboard(
                    pattern: "/umbraco/backoffice/hangfire",
                    options: new DashboardOptions() { 
                        Authorization = new[] { new UmbracoAuthorizationFilter() }
                    }).RequireAuthorization(Constants.System.HangfireDashboard);
        }).UseHangfireDashboard()
    });
});
```

(P.S. added pull request)

**Comment by digbyswift** [comment]
@nul800sebastiaan @Axendo I have had exactly the same issue in the last week but deploying to a Windows Azure app service. The fix above worked a charm for me. I was using Hangfire 1.7.27 on Umbraco 9.1.0.

**Comment by nul800sebastiaan** [comment]
Forgot to close this one! This was fixed in a different way with PR #9 - should work in v1.0.1.
